create-release.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o pipefail
+
+die() { set +v; echo "$*" 1>&2 ; exit 1; }
+
+git diff --quiet || die 'Uncommitted changes: Stash or commit before pushing.'
+
+(( $# == 1 )) || die "Requires one argument, the NPM version type (major | minor | patch); Instead got $#."
+
+if ! command -v gh &> /dev/null
+then
+    die "Requires the GitHub CLI (gh)."
+fi
+
+if ! command -v aws &> /dev/null
+then
+    die "Requires the AWS CLI (aws)."
+fi
+
+AWS_ACCT=$(aws iam list-account-aliases --query 'AccountAliases[0]')
+if [[ "$AWS_ACCT" != "gehlenborglab" ]]; then
+    read -p "You are not logged into the lab account in the AWS CLI. Are you sure you want to make a release from this account? [y/n]" -n 1 -r
+    echo # move to a new line
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
+        echo "Logged into the lab account"
+    else
+        exit
+    fi
+fi
+
+BRANCH=`git rev-parse --abbrev-ref HEAD`
+if [[ "$BRANCH" != "master" ]]; then
+    read -p "You are not on the master branch. Are you sure you want to make a release from this branch? [y/n]" -n 1 -r
+    echo # move to a new line
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
+        echo "Continuing the release"
+    else
+        exit
+    fi
+fi
```

Comment:
I would vote for similarly not allowing this unless there is good reason.

Reply:
I mainly needed this to test on this non-master branch for the PR. I think it could be helpful in the future if anyone wants to try updating this script on a new branch.